examples/dotnet/SslPutGet/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required(VERSION 3.10)
+
+project(dotnet-SslPutGet LANGUAGES NONE)
+
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/CMakeLists.txt.in ${CMAKE_CURRENT_BINARY_DIR}/CMakeLists.txt @ONLY)
+
+install(FILES
+  ${CMAKE_CURRENT_BINARY_DIR}/CMakeLists.txt
+  Program.cs
+  README.md
+  startserver.ps1
+  stopserver.ps1
+  DESTINATION examples/dotnet/SslPutGet)
```

Comment:
what? no whitespace here...

Reply:
you have a `.in` file, should this file be in repo? as well

Reply:
The ssl example has additional requirements the standard example template doesn't cover, so it has its own. That's why the root cmake file uses add_directory instead of add_example, the cmake file in the ssl example directory populates the template in the example install directory. It has to be done this way because there are variables available when installing examples that are not available when building examples.